yang-module-versioning/draft-ietf-netmod-yang-module-versioning.txt

```diff
@@ -811,267 +696,169 @@ Internet-Draft    Updated YANG Module Revision Handling     October 2023
+            2019-01-01
                 |
-            2019-02-01                 <- 2.0.0
+            2019-02-01
                 |      \
-            2019-03-01  \              <- 3.0.0
+            2019-03-01  \
                 |        \
-                |       2019-04-01     <- 2.1.0
+                |       2019-04-01
                 |           |
-                |       2019-05-01     <- 2.2.0
-                |
-            2019-06-01                 <- 3.1.0
+            2019-05-01      |
+                            |
+                        2019-06-01
 
 4.1.1.1.  Example 1
 
    This example recommends module revisions for import that match, or
    are derived from the revision 2019-02-01.  E.g., this dependency
    might be used if there was a new container added in revision
    2019-02-01 that is augmented by the importing module.  It includes
-   revisions/labels: 2019-02-01/2.0.0, 2019-03-01/3.0.0,
-   2019-04-01/2.1.0, 2019-05-01/2.2.0 and 2019-06-01/3.1.0.
-
-   import example-module {
-     rev:recommended-min 2019-02-01;
-   }
-
-   Alternatively, the first example could have used the revision label
-   "2.0.0" instead, which selects the same set of revisions/labels.
-
-
-
-Wilton, et al.           Expires April 19, 2024                [Page 16]
-
-Internet-Draft    Updated YANG Module Revision Handling     October 2023
-
+   the following revisions: 2019-02-01, 2019-03-01, 2019-04-01,
+   2019-05-01 and 2019-06-01.
 
    import example-module {
-     rev:recommended-min 2.0.0;
+     rev:recommended-min-date 2019-02-01;
    }
 
 4.1.1.2.  Example 2
 
-   This example recommends module revisions for import that are derived
-   from 2019-04-01 by using the revision label 2.1.0.  It includes
-   revisions/labels: 2019-04-01/2.1.0 and 2019-05-01/2.2.0.  Even though
-   2019-06-01/3.1.0 has a higher revision label number than
-   2019-04-01/2.1.0 it is not a derived revision, and hence it is not a
-   recommended revision for import.
-
-   import example-module {
-     rev:recommended-min 2.1.0;
-   }
-
-4.1.1.3.  Example 3
-
-   This example recommends module revisions for import that are derived
-   from either 2019-04-01 or 2019-06-01.  It includes revisions/labels:
-   2019-04-01/2.1.0, 2019-05-01/2.2.0, and 2019-06-01/3.1.0.
-
-   import example-module {
-     rev:recommended-min 2019-04-01;
-     rev:recommended-min 2019-06-01;
-   }
-
-5.  Updates to ietf-yang-library
-
-   This document updates YANG 1.1 [RFC7950] and YANG library [RFC8525]
-   to clarify how ambiguous module imports are resolved.  It also
-   defines the YANG module, ietf-yang-library-revisions, that augments
-   YANG library [RFC8525] with revision labels and two leafs to indicate
-   how a server implements deprecated and obsolete schema nodes.
-
-5.1.  Resolving ambiguous module imports
+   This example recommends module revisions for import that match, or
+   are derived from the revision 2019-04-01.  It includes the following
```

Comment:
Need to remove derived from in this sentence.

Reply:
Maybe but if we kept those paragraphs describing what derived from means then leaving that term is probably OK IMO.

Reply:
But the recommended-min-date logic does not use "derived from" in its decision.

Reply:
Good point. I looked at this more closely and I think you're right that in Example 1 and Example 2 (sections 4.1.1.1 and 4.1.1.2) we should indeed remove the work "derived".  Maybe rework like this:

This example recommends module revisions for import that match, or are later than the revision 2019-02-01.

Reply:
I had already changed it a few days ago to the following. I'll keep it as is for now and can change subsequently (it's editorial anyway).
   This example recommends module revisions for import whose revision
   date is or comes after 2019-02-01.